crates/hyperdrive-math/src/long/close.rs

```diff
@@ -36,6 +36,13 @@ impl State {
         normalized_time_remaining: F,
     ) -> FixedPoint {
         let bond_amount = bond_amount.into();
+
+        if bond_amount < self.config.minimum_transaction_amount.into() {
+            // TODO would be nice to return a `Result` here instead of a panic.
```

Comment:
Agreed. If we don't implement them in this PR, can we add an issue for these so it can be prioritized in the backlog?

Reply:
Agreed; leave it as a TODO & make an issue. I'll handle it, I already fixed a few in targeted_long PR and I think I'm going to just do all of them.